src/sync/installation.ts

```diff
@@ -1,4 +1,7 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
+import { intersection } from "lodash";
+import IORedis from "ioredis";
```

Comment:
I like having node module imports up the top so I moved these here to get opinions....
`import { intersection } from "lodash"; `is new tho